lib/tasks/dev.rake

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+namespace :dev do
+  require 'factory_bot_rails'
+  require 'faker'
+  desc 'Generate test data'
+  task :generate_data, [:number] => :environment do |_, args|
+    setargs(args)
+    language_codes = %w[vn en jp lao]
+    programming_language_codes = %w[ruby php java net assembly]
+    @args[:number].times do
+      email = Faker::Internet.email
+      developer = FactoryBot.create(:developer, email: email)
+      languages = language_codes.sample(rand(4))
+      languages.each do |f|
+        language = Language.find_or_create_by(code: f)
```

Comment:
There's only four languages, `%w[vn en jp lao]`.
The number of developers is 100 by default.
It looks very inefficient that you have to run this query every time you create a developer.